src/Test.UnitTests.Sarif.Multitool.Library/ExportValidationRulesDocumentationCommandTests.cs

```diff
@@ -14,18 +14,27 @@ public class ExportValidationRulesDocumentationCommandTests
         [Fact]
         public void BuildRule_GeneratesExpectedMarkdown()
         {
+            var tests = new Dictionary<string, SarifValidationSkimmerBase>
+            {
+                { "Test1.md", new TestRule1() },
+                { "Test2.md", new TestRule2() },
+                { "Test3.md", new TestRule3() }
```

Comment:
>Test3 [](start = 19, length = 5)

Use descriptive test file names like "FullMarkdownRuleDescription.md", "NoDescription.md", "StandardMessageStringKey.md", "NonStandardMessageStringKey.md", and so on.

Also: change only one thing in each test. For Test2 is different from Test1 in _two_ ways: it changes from full description to short description, and from a standard message string key to a non-standard key. Let's have separate test cases for each "difference".